server/chart/chart.test.js

```diff
@@ -0,0 +1,29 @@
+import MongodbMemoryServer, { MongoMemoryServer } from "mongodb-memory-server";
```

Comment:
There seem to be quite a few unused import statements/variables in this file that should be removed.